roles/nf-core-biocontainer-temp/tasks/main.yml

```diff
@@ -16,12 +16,14 @@
 
 - name: Create temp directories for download
   file:
-    path: "{{ ngi_containers }}/download"
+    path: "{{ item }}"
     state: directory
+  with_items:
+    - "{{ ngi_containers }}/download"
+    - "{{ ngi_containers }}/download/biocontainers"
```

Comment:
According to the docs only this path is needed. `download` should be created if it does not exist. But I'm looking at new docs, so maybe it's not supported in the ansible version we are currently using.

Reply:
Updated it! We'll have the ansible v2.9 changes on the same branch, so we can go with what it says in the latest docs.